.github/workflows/dockerhub-master.yml

```diff
@@ -0,0 +1,25 @@
+name: DockerHub (Master)
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-tag-push-master:
+    name: build-tag-push-master
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      
+      - name: build-tag-push
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          dockerfile: docker/Dockerfile
+          cache_froms: buildpack-deps:stable-curl
+          repository: xaynetwork/xaynet
+          tag_with_ref: true
+          tags: development
```

Comment:
This is not only tagging that image built on this merge-to-master with `latest` but also with `development`, since that's also what we use on the Development Kubernetes Cluster for the coordinator.